validator/src/main.rs

```diff
@@ -239,8 +247,16 @@ fn main() {
     if let Some(paths) = matches.value_of("accounts") {
         validator_config.account_paths = Some(paths.to_string());
     }
-    if let Some(paths) = matches.value_of("snapshot_path") {
-        validator_config.snapshot_path = Some(paths.to_string());
+    if let Some(snapshot_path) = matches.value_of("snapshot_path") {
+        let snapshot_package_output_path = matches.value_of("snapshot_package_output_path").expect(
+            "Snapshots require both a snapshot_path and snapshot_package_output_path, missing snapshot_package_output_path"
```

Comment:
clap can do this for you, and in a more user-friendly way: https://docs.rs/clap/2.33.0/clap/struct.Arg.html#method.requires